### ISSUE TITLE
Questions regarding psme and tabsyndex as loss function

### ISSUE THREAD

**Issue by SvenGroen** [vikram2000b/tabsyndex#1]
Hello,

I am using tabsyndex for my current project and was wondering if you could help me out:

My first question is: in your code where you calculate the psme score, you fit the regression model on the same part of the data that you use for prediction. 
Don't you have to train – test split the data in this scenario first? If not, why not?

My data synthesis model also had a pmse score close to 0 like you observed in your paper and I am trying to figure out why. 
My first thought was, that the regression model just overfits and learns which rows are fake and which ones are real. 
But on the other hand, if this would be the case the psme score wouldn't be so high if we input 2 subsets from the same dataset like you did in your example.ipynb.

My second question is regarding using the tabsyndex score as a loss function.
In order to calculate the score you need 2 data frames, so you have to sample some synthetic data first.
Do you remember how many data rows you sampled before calculating the score? 
With my model the sampling process is computationally pretty expensive and using the sim score as a loss would increase overall training time.

Cheers,
Sven
